src/components/LibrariesListContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import LibrariesList from "./LibrariesList";
+import { Store } from "redux";
+import { State } from "../reducers/index";
+import { LibrariesData, LibraryData } from "../interfaces";
+import { connect } from "react-redux";
+import ActionCreator from "../actions";
+
+export interface LibrariesListContainerContext {
+  store: Store<State>;
+  csrfToken: string;
+}
+
+export interface LibrariesListContainerStateProps<T> {
+  store?: Store<State>;
+}
+
+export interface LibrariesListContainerDispatchProps<T> {
+  fetchData?: () => Promise<T>;
+}
+
+export interface LibrariesListContainerOwnProps {
+  store?: Store<State>;
+  csrfToken?: string;
+}
+
+export interface LibrariesListContainerProps<T> extends LibrariesListContainerDispatchProps<T>, LibrariesListContainerOwnProps {}
```

Comment:
Since this isn't a connected component I'm confused about what these props are doing. Do you even need them?

Reply:
That...is a very good point.  I will get rid of them :)